.github/workflows/gpu-ci-docker.yaml

```diff
@@ -0,0 +1,30 @@
+name: GPU CI
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+jobs:
+  gpu-ci-docker:
+    runs-on: linux-amd64-gpu-p100-latest-1
+
+    container:
+      image: nvcr.io/nvstaging/merlin/merlin-ci-runner:latest
+      credentials:
+        username: ${{ secrets.SVC_DOCKER_USER }}
+        password: ${{ secrets.SVC_DOCKER_TOKEN }}
+      env:
+        NVIDIA_VISIBLE_DEVICES: ${{ env.NVIDIA_VISIBLE_DEVICES }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get Branch name
+        id: get-branch-name
+        uses: NVIDIA-Merlin/.github/actions/branch-name@6f0539fba24f60da2aee63c5925bee7cee3206e3
+      - name: Run tests
+        run: |
+          merlin_branch="${{ steps.get-branch-name.outputs.branch }}"
+          MERLIN_BRANCH=$merlin_branch COMPARE_BRANCH=$merlin_branch \
```

Comment:
These env vars are not required here. 

- `MERLIN_BRANCH` is used in other places where we need to install other merlin packages from the equivalent branch ( release branch or development branch). In this `test-gpu` tox environment we only install the core package.
  - the `get-branch-name` step can also be removed here too since we won't use it if `MERLIN_BRANCH` is removed
- `COMPARE_BRANCH` is something currently only used in Merlin Models to inspect the diff and label parts as changed or unchanged (so that tests using TensorFlow only run if the tensorflow code has changed).